src/charm.py

```diff
@@ -105,7 +109,7 @@ def __init__(self, *args):
             self,
             scrape_configs=[
                 {
-                    "scheme": "https",
+                    "scheme": "http",
```

Comment:
Why are we changing this in this PR?

Reply:
Https was never supported in this charm, `https` was used all over the place but never actually worked. We'll be adding https support when we add TLS certs part of this charm.

Reply:
Ok, thanks for clarifying!